products/tomcat.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Apache Tomcat
+permalink: /tomcat
+iconSlug: apache
+layout: post
+releasePolicyLink: https://tomcat.apache.org/whichversion.html
+changelogTemplate: https://dlcdn.apache.org/tomcat/tomcat-__RELEASE_CYCLE__/v__LATEST__/RELEASE-NOTES
+category: server-app
+activeSupportColumn: false
+command: ./bin/version.sh
+releaseColumn: true
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+releases:
+  - releaseCycle: "10"
+    release: 2020-12-08
+    eol: false
+    latest: "10.0.18"
+    link: https://tomcat.apache.org/download-10.cgi
+  - releaseCycle: "9"
+    release: 2017-10-04
+    eol: false
+    latest: "9.0.60"
+    link: https://tomcat.apache.org/download-90.cgi
+  - releaseCycle: "8"
+    release: 2014-02-02
+    eol: false
+    latest: "8.5.76"
+    link: https://tomcat.apache.org/download-80.cgi
+  - releaseCycle: "7"
+    release: 2011-01-13
+    eol: 2021-03-31
+    latest: "7.0.109"
+  - releaseCycle: "6"
+    release: 2006-10-21
+    eol: 2016-12-31
+    latest: "6.0.53"
+  - releaseCycle: "5"
+    release: 2003-09-06
+    eol: 2012-09-30
+    latest: "5.5.36"
+  - 
+---
+> The Apache Tomcat® software is an open source implementation of the [Jakarta Servlet](https://projects.eclipse.org/projects/ee4j.servlet), [Jakarta Server Pages](https://projects.eclipse.org/projects/ee4j.jsp), [Jakarta Expression Language](https://projects.eclipse.org/projects/ee4j.el), [Jakarta WebSocket](https://projects.eclipse.org/projects/ee4j.websocket), [Jakarta Annotations](https://projects.eclipse.org/projects/ee4j.ca) and [Jakarta Authentication specifications](https://projects.eclipse.org/projects/ee4j.authentication). These specifications are part of the [Jakarta EE platform](https://projects.eclipse.org/projects/ee4j.jakartaee-platform).
+
+The update and release strategy is _complex_. For release policy information, see <https://tomcat.apache.org/whichversion.html>.
```

Comment:
Complexity of the policy is not a good reason for us to omit it entirely. While upstream projects might have to cover everything, we can reduce complexity in a lot of ways - we don't need to cover historical policies for EoL releases, and we only cover stable releases.

@bendem Wanna give this a shot? Or I can try in a bit.